src/components/InfoPanel.vue

```diff
@@ -122,7 +122,7 @@
           </div>
           <div>
             <!-- TODO handle null lat/lng? -->
-            <a :href="`https://www.google.com/maps/search/?api=1&query=${location.fields['Latitude']},${location.fields['Longitude']}`" target="_blank">
+            <a :href="`https://www.google.com/maps/dir/?api=1&destination=${location.fields['Latitude']},${location.fields['Longitude']}`" target="_blank">
```

Comment:
Do we want to use the clicked location on the map as a starting point? This currently defaults to the user's current location (if available, otherwise it will prompt for a starting point).